priv/repo/migrations/20180618202035_create_hardware_identifiers_table.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Console.Repo.Migrations.CreateHardwareIdentifiersTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:hardware_identifiers, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :token, :binary, null: false
+      timestamps()
+    end
+
+    create unique_index(:hardware_identifiers, [:token])
+
+    alter table(:gateways) do
+      add :hardware_identifier_id, references(:hardware_identifiers, on_delete: :nothing, type: :binary_id), null: false
```

Comment:
does `on_delete: :nothing` make sense here?

Reply:
made the fix. Also it seems the on_delete: :delete_all in hardware_identifier.ex does the same thing.